linux_os/guide/system/software/integrity/software-integrity/aide/aide_disable_silentreports/rule.yml

```diff
@@ -0,0 +1,49 @@
+documentation_complete: true
+
+prodtype: ubuntu2004
+
+title: 'AIDE must notify personnel if baseline configurations are altered'
+
+description: |-
+    The operating system file integrity tool must be configured to notify designated personnel of any changes to configurations.
+
+rationale: |-
+    Detecting changes in the system can help avoid unintended, and negative consequences 
+    that could affect the security state of the operating system
+
+severity: medium
+
+references:
+    disa: 'CCI-001744,CCI-002702'
```

Comment:
please remove the single quotes

Reply:
Single quotes removed https://github.com/ComplianceAsCode/content/pull/10738/commits/c7c9463317a6570a08df6710bd1d98692ea3ae11